debugQrCode: false
qrcodeSharpenBlur: 5
qrcodeSharpenWeight: 0.8

